XIPs/xip-reaction-content-type.md

```diff
@@ -0,0 +1,60 @@
+---
+xip: 
+title: Reaction Content Type
+description: Content type for reactions
+author: Kunal Mondal
+status: Draft 
+type: Standards Track
+category: XRC
+created: 2023-03-24
+---
+
+## Abstract
+
+This XRC proposes a new content type for messages that enriches the messaging experience and let's users express themselves using emojis. It attempts to include only the bare minimum for clients to determine how to display such messages in order to provide flexibility for more rich types in the future. 
+
+## Motivation
+
+When it comes to having a comfortable conversation, reacting to messages with emojis is necessary as it gives users the freedom to express their messages in form of emoji. In order to provide users a real messaging platform, XMTP needs to provide this capability.
+
+## Specification
+
+Proposed content type:
+
+```js
+{
+    authorityId: "xmtp.org",
+    typeId: "reaction",
+    versionMajor: 1,
+    versionMinor: 0,
+}
+```
+
+The reaction MUST include the following parameters:
+
+Adding info about the message we’re reacting to (it’s message id, sender address, message content, content type id and the reaction). 
+
+```ts
+{
+    messageId: string
+    senderAddress: string
```

Comment:
> Yes right. We can use the senderAddress from the XMTP message itself instead of relying on this. Should I remove it and commit ?

Ya you can remove this and commit.

Reply:
> > Yes right. We can use the senderAddress from the XMTP message itself instead of relying on this. Should I remove it and commit ?
> 
> Ya you can remove this and commit.

Okay sure.